packages/synapse-react-client/src/utils/functions/DateFormatter.ts

```diff
@@ -1,16 +1,24 @@
 import SynapseClient from '../../synapse-client'
 import dayjs, { Dayjs } from 'dayjs'
 import utc from 'dayjs/plugin/utc'
+import advancedFormat from 'dayjs/plugin/advancedFormat'
+import localizedFormat from 'dayjs/plugin/localizedFormat'
 
 dayjs.extend(utc)
+dayjs.extend(advancedFormat)
+dayjs.extend(localizedFormat)
 
-export function formatDate(
-  time: Dayjs,
-  pattern: string = 'M/D/YYYY h:mm A',
-): string {
+export function formatDate(time: Dayjs, pattern?: string): string {
+  if (!pattern) {
+    // By default, only show timezone if using UTC time
+    pattern = 'M/D/YYYY h:mm A'
+    if (SynapseClient.getUseUtcTimeFromCookie()) {
+      pattern += ' z'
+    }
+  }
```

Comment:
Before, we would always append `' UTC'` if the UTC cookie was set.

Now, only show the timezone by default if there's no pattern override.

This will fix a few cases where we're just showing a date (`'01/01/2024'`). It usually doesn't make sense to append a timezone if you're showing a date without a time!


---

I think it would be good to change our default pattern to use [localized formats](https://day.js.org/docs/en/display/format#localized-formats), but that probably deserves some dedicated time to ensure we use it consistently and don't mix date formats throughout the app.

Reply:
Using localized formats is a great idea - is there a Jira ticket to track?

Reply:
I'll add some more context to this old ticket and move it to dev triage! https://sagebionetworks.jira.com/browse/SWC-5410